man/varlinkctl.xml

```diff
@@ -123,14 +123,14 @@
         <term><command>introspect</command> <replaceable>ADDRESS</replaceable> <replaceable>INTERFACE</replaceable></term>
 
         <listitem><para>Show interface definition of the specified interface provided by the specified
-        service. Expects a service address in the formats described above and a Varlink interface
+        service. Expects a service address in one of the formats described above and a Varlink interface
         name.</para>
 
         <xi:include href="version-info.xml" xpointer="v255"/></listitem>
       </varlistentry>
 
       <varlistentry>
-        <term><command>call</command> <replaceable>ADDRESS</replaceable> <replaceable>METHOD</replaceable> [<replaceable>ARGUMENTS</replaceable>]</term>
+        <term><command>call</command> <replaceable>ADDRESS</replaceable> <replaceable>METHOD</replaceable> [<replaceable>PARAMETERS</replaceable>]</term>
```

Comment:
I think this particular change is wrong.

Normally the parameter/argument distinction is that parameters are the names declared at the function definition that will be used to refer to passed values, while arguments are the expressions located in function call syntax that correspond to the parameters (or sometimes the values that those expressions resolve to. The definition varies by source.)

Since here we are specifying values as part of a function call these are indeed arguments. I believe the correct fix here is instead to change PARAMETERs to ARGUMENTS up in the cmdsynopsis element.

Reply:
Thank you, your explanation was very detailed. Already fixed.